sites/fast-creator/app/web-components/index.tsx

```diff
@@ -121,8 +126,8 @@ export function getColorPickerControls(
     updateHandler: (updatedData: { [key: string]: unknown }) => void
 ): StandardControlPlugin[] {
     return [
-        getColorPickerControl("background-color", updateHandler),
-        getColorPickerControl("accent-base-color", updateHandler),
+        getColorPickerControl("fill-color", updateHandler),
+        getColorPickerControl("accent-color", updateHandler),
```

Comment:
This isn't a realized property in the new tokens, but if you're mapping to the palette I suggest we keep the "base" aspect in this name.